front/lib/resources/membership_resource.ts

```diff
@@ -337,17 +331,14 @@ export class MembershipResource extends BaseResource<MembershipModel> {
       { where: { id: membership.id }, transaction }
     );
 
-    void ServerSideTracking.trackRevokeMembership({
-      user: user.toJSON(),
-      workspace,
+    return new Ok({
       role: membership.role,
       startAt: membership.startAt,
       endAt,
     });
-
-    return new Ok(undefined);
   }
 
+  // Caller of this method should call `ServerSideTracking.trackUpdateMembershipRole`.
```

Comment:
Using proper JSDoc this will be displayed on hover and in auto suggestion.

Reply:
(updating)